minio-client/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	minio "github.com/minio/minio-go"
+)
+
+var (
+	accessKeyID              = os.Getenv("ACCESS_KEY")
+	secretAccessKey          = os.Getenv("SECRET_KEY")
+	serviceName              = os.Getenv("SERVICE_NAME")
+	namespace                = os.Getenv("NAMESPACE")
+	port                     = os.Getenv("PORT")
+	livenessCheckInterval, _ = strconv.Atoi(os.Getenv("LIVENESS_CHECK_INTERVAL"))
+	livenessRetryCount, _    = strconv.Atoi(os.Getenv("LIVENESS_RETRY_COUNT"))
+	mode                     = os.Getenv("MODE")
+	capacity, _              = strconv.ParseFloat(os.Getenv("CAPACITY"), 32)
+)
+
+func main() {
+	fileName := "bigfile.test"
+	bucketName := "test-bucket" + "-" + fmt.Sprintf("%d", time.Now().Unix())
+	endpoint := serviceName + "." + namespace + ".svc.cluster.local:" + port
+	useSSL := false
+
+	// Initialize minio client object.
+	minioClient, err := minio.New(endpoint, accessKeyID, secretAccessKey, useSSL)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// log.Printf("%#v\n", minioClient) // minioClient is now setup
+
+	createBucket(minioClient, bucketName, "us-east-1")
```

Comment:
should this region be a var too? or is it a dummy entity?  (sorry, not too sure of minio internals)

Reply:
Region field is required to differentiate between the buckets. In this case single bucket is getting created which is getting loaded and unloaded every time with some random object. Region here does not really makes any difference here.